src/components/HeroSection/HeroSection.test.js

```diff
@@ -0,0 +1,8 @@
+import HeroSection from "./HeroSection.js";
+import React from "react";
+import renderer from "react-test-renderer";
+
+it("renders correctly", () => {
+  const tree = renderer.create(<HeroSection />).toJSON();
```

Comment:
please pass some props to the component in order to see if it renders things correctly or not

Reply:
I will do. Thanks